mmcv/ops/point_sample.py

```diff
@@ -1,9 +1,95 @@
 # Modified from https://github.com/facebookresearch/detectron2/tree/master/projects/PointRend  # noqa
 
+from os import path as osp
+
 import torch
 import torch.nn as nn
 import torch.nn.functional as F
 from torch.nn.modules.utils import _pair
+from torch.onnx.operators import shape_as_tensor
+
+from mmcv.ops import get_onnxruntime_op_path
```

Comment:
We could move this import to line 90(inside is_in_onnx_export_without_custom_ops) to avoid import the module when there is no onnx exporting.

Reply:
Fixed.